### PR TITLE
update table name and add new column

### PR DESCRIPTION
**Pull request by ssilwal29**
```
▶ git commit -m "one more null edge case"
Check Yaml...........................................(no files to check)Skipped
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
black....................................................................Passed
build-docker-run-tests...................................................Passed
```

### CODE REVIEW

File: airflow_prometheus_exporter/prometheus_exporter.py [hunk 1]
```diff
@@ -603,13 +619,13 @@ def collect(self):
...
-                    task.instant_slack_alert or "none",
-                    task.alert_classification or "none",
-                    task.sla_time or "none",
+                    task.state or "missing",
+                    task.cadence or "missing",
+                    task.severity or "missing",
+                    task.alert_target or "missing",
+                    task.alert_external_classification or "missing",
+                    task.alert_report_classification or "missing",
+                    task.sla_time or "missing",
```
Comment (jsu) [inline airflow_prometheus_exporter/prometheus_exporter.py]: Should we make a constant for "missing"?

File: docker/airflow.sh [hunk 2]
```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 AF_HOME="/usr/local/airflow"
-sqlite3 "${AF_HOME}/airflow.db" "create table gap_dag_tag(
+sqlite3 "${AF_HOME}/airflow.db" "create table delay_alert_metadata(
     dag_id varchar(250) PRIMARY KEY,
 	task_id varchar(250),
 	cadence varchar(250),
-	severerity varchar(250),
+	severity varchar(250),
 	alert_target varchar(250),
```
Comment (jsu) [inline docker/airflow.sh:8]: Should we fix the indent here?

**Review by jsu — APPROVED** [review]

File: airflow_prometheus_exporter/prometheus_exporter.py (hunk 1 above)
Comment (ssilwal29) [reply airflow_prometheus_exporter/prometheus_exporter.py]: 👍

File: docker/airflow.sh (hunk 2 above)
Comment (ssilwal29) [reply docker/airflow.sh:8]: 👍

**Pull request merged**